crates/tedge_command_config/tests/test_set_command.rs

```diff
@@ -0,0 +1,13 @@
+fn set_command_fails_given_device_id() {
```

Comment:
no CLI parsing involved here. we want to test one and only one thing at a time. Not CLI-parsing + command execution.